crates/uv-python/src/downloads.rs

```diff
@@ -508,3 +511,18 @@ impl Display for ManagedPythonDownload {
         write!(f, "{}", self.key)
     }
 }
+
+fn installation_has_build_info(p: &Path) -> bool {
+    let mut has_install = false;
+    let mut has_build = false;
+    if let Ok(dir) = p.read_dir() {
+        for entry in dir.flatten() {
+            match entry.file_name().to_str() {
+                Some("install") => has_install = true,
```

Comment:
I might suggest just checking these directly? We know where they would be, right? We shouldn't need to do a full directory scan.

Reply:
(As in: `p.join("install").is_dir()`.)